packages/zowe-explorer-api/src/logger/IZoweLogger.ts

```diff
@@ -0,0 +1,85 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the *
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at *
+ * https://www.eclipse.org/legal/epl-v20.html                                      *
+ *                                                                                 *
+ * SPDX-License-Identifier: EPL-2.0                                                *
+ *                                                                                 *
+ * Copyright Contributors to the Zowe Project.                                     *
+ *                                                                                 *
+ */
+
+import * as loggerConfig from "../log4jsconfig.json";
+import * as path from "path";
+import { Logger } from "@zowe/imperative";
+
+export enum MessageSeverityEnum {
```

Comment:
Is it possible to transfer this to CONSTANTS file? or it is important that this enum is defined here?

Reply:
This is in the API, we don't have a constants file there I think?

Reply:
@jellypuno

Reply:
ahhhh. we have one in ZE package. o well. nevermind then